front/app/containers/EventsPage/index.tsx

```diff
@@ -29,13 +29,7 @@ export default () => {
   const locale = useLocale();
 
   if (isNilOrError(locale)) return null;
-
-  if (!eventsPageEnabled) {
-    // This redirects to the 'page not found' page while keeping the requested URL
-    clHistory.replace(`/${locale}/*`);
-    window.history.replaceState(null, '', `/${locale}/events`);
-    return null;
-  }
+  if (!eventsPageEnabled) return redirectToNotFoundPage(locale, 'events');
```

Comment:
Did you check if it works in your browser? :)

Reply:
Yup, works on my side